src/util/converter/SearchStringConverter.js

```diff
@@ -0,0 +1,106 @@
+function parseWildcard(searchString) {
+    if (!searchString.includes("*")) {
+        return `^.*${searchString}.*$`;
+    }
+    return `^${searchString.replace(/\*+/g, ".*")}$`;
+}
+
+function parseNot(searchString) {
+    searchString = searchString.trim();
+    if (searchString.startsWith("!")) {
+        return `(?:(?!${parseWildcard(searchString.slice(1))})^.*$)`;
+    }
+    return parseWildcard(searchString);
+}
+
+function parseAnd(searchString) {
+    const result = searchString.split("&").map(parseNot);
+    if (result.length > 1) {
+        return `(?=${result.join(")(?=")}).*`;
+    }
+    return result[0];
+}
+
+function parseOr(searchString) {
+    return searchString.split("/").map(parseAnd).join("|");
```

Comment:
Muss das "/" als ODER sein? War das ne Anforderung? Problem aus meiner Sicht ist, das Behörden dieses Zeichen wirklich benutzen in bestimmten Feldern. Wenn dann mal danach gefiltert werden soll, könnte das unschön sein

Reply:
thomas meinte ja, ich wollte eigentlich das "|" verwenden. vielleicht könnten wir nochmal anführen, dass das "/" in daten verwendet werden könnte

Reply:
es wird das "/". im zweifel können wir irgendwann noch anführungszeichen einführen um ganze zeichenfolgen zu matchen, ohne "&" und "/" zu parsen